bpfd-operator/apis/v1alpha1/shared_types.go

```diff
@@ -102,3 +102,178 @@ type ImagePullSecretSelector struct {
 	// Namespace of the secret which contains the credentials to access the image repository.
 	Namespace string `json:"namespace"`
 }
+
```

Comment:
Duplicated comment

Reply:
Fixed in 6eb831f66bb60553470cadf38b50c0a1e78b3760 :+1: